resources/views/auth/login.blade.php

```diff
@@ -1,7 +1,7 @@
 <x-guest-layout>
     <x-jet-authentication-card >
         <x-slot name="logo">
-        <img src="https://www.csu.qc.ca/wp-content/uploads/2020/01/CSU_logo_Colored.png" alt= "Csu logo">
+        <img src="/storage/logos/app_logo.png" alt= "Csu logo">
```

Comment:
Can you use the application logo component here instead of an <img> :)

Reply:
1. Add <application-logo> instead of this <img> tag here.
2. Move the <img> tag inside the ApplicationLogo component replacing the svg part with the <img> that has a src path.

Is that what you mean?

Reply:
no we have a component that already places the logo. The reason is that we have the option in settings to change this photo. the component will either display the default or use the one that was uploaded in teh settings

Reply:
i do believe that is what he is talking about

Reply:
1. yes not 2. 

The application logo component already has the logic to display the correct image

Reply:
true. if you still have the svg in there than you may need to pull/rebase your branch

Reply:
ok i was wrong, I thought we had put the logic for the logo in the in the component. 

So yes @danseremet if you could update https://github.com/CSU-Booking-Platform/application/blob/main/resources/js/Jetstream/ApplicationLogo.vue to have the logic to display the application logo (either default CSU logo, or the logo set via settings) that would be great

Reply:
and move the component to js/Components since its not a jetstream default component anymore.

Reply:
The logic should already be there. Just replace SVG with the default image path. You shouldn't need to affect any backend code for this pull

Reply:
You need $page.app_logo on login page. I can't get it from this file, so I'm using img.
A new PR is coming so I'm closing this one.